portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase

```diff
@@ -1710,4 +1710,62 @@ definition {
 		}
 	}
 
+	@description = "This is for LPS-133450. View the master page reverts to default. Test reported in bug LPS-131982."
```

Comment:
Should be This is a test for LPS-131982. Change master of content page with HTML title.